src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,122 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(nullable = false)
+    private String isbn;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column
+    private String genre;
+
+    @Column(nullable = false)
```

Comment:
Following the line of my previous comment, instead of using @Column(nullable = false) you can use directly @NotNull.